tox.ini

```diff
@@ -20,5 +28,13 @@ deps =
     pylint
     flake8
 commands =
-    flake8 --max-complexity=10 pytest_datafiles.py tests/test_pytest_datafiles.py
-    pylint pytest_datafiles.py tests/test_pytest_datafiles.py
+    flake8 --max-complexity=10 pytest_datafiles.py tests/
+    pylint pytest_datafiles.py tests/
```

Comment:
The examples should not be linted. It adds unnecessary complication, as can be seen in at least one comment to disable a pytlint linter in an example file.

Reply:
Linting the examples should still be done IMHO, but I'll make it optional.

Reply:
via the pytester route, it's part of the regular linting without becoming awkward,